scripts/prReleaseNotes.js

```diff
@@ -120,17 +133,59 @@ function generateNotes(PRs) {
   addTitle('OTHERS');
   others.forEach(addEntry);
 
+  // Silent
+  addTitle('// Silent - these PRs did not have a changelog or were left out for some other reason, is it on purpose?');
+  silentPRs.forEach(addEntry);
+
   fs.writeFileSync(`${process.env.HOME}/Downloads/uilib-release-notes_${NEW_VERSION}.txt`, releaseNotes, {
     encoding: 'utf8'
   });
 
   console.log(`\x1b[1m\x1b[32m✔\x1b[0m \x1b[32muilib-release-notes.txt was successfully written to ${process.env.HOME}/Downloads\x1b[0m \x1b[1m\x1b[32m✔\x1b[0m`);
 }
 
+function isSilent(pr) {
+  if (!pr.info.changelog) {
+    return true;
+  } else {
+    const changelog = pr.info.changelog.toLowerCase();
+    if (changelog === 'none' || changelog === 'n/a') {
+      return true;
+    }
+  }
+
+  return false;
+}
+
 function addTitle(title) {
   releaseNotes += `\n\n${title}\n\n`;
 }
 
 function addEntry(pr) {
-  releaseNotes += `• ${pr.info.changelog || pr.title} (#${pr.number}) \n`;
+  const log = pr.info.changelog || pr.title;
+  const requester = getRequester(pr);
```

Comment:
We're not using Jira tickets here...

Reply:
It's true, but when thinking about it, it makes the code the same in both places.
Maybe I'll make them even closer and use this in the private one.
WDYT?

Reply:
Yes, I think this should be applied only in the private notes. Here we can add a section for linking issues or other notes if we want

Reply:
Why should it only be in the private repo? It'll not be straightforward to make it pluggable.

Reply:
We're not attaching Jira links in the public repo for security measures, so this section only relevant to the private repo

Reply:
Do you mean to keep one file in public and run it from the private as well (instead of keeping the two copies)?